Druid/PCGArrayNode.extension.st

```diff
@@ -0,0 +1,7 @@
+Extension { #name : #PCGArrayNode }
+
+{ #category : #'*Druid' }
+PCGArrayNode >> maxCompiledPrimitives [
```

Comment:
Why on array ?

Reply:
It answers the number of "definitions" (elements) that specify a primitive in the Array of primitives table. This is the parameter of the "from:" in #initializePrimitiveTable

Reply:
Mmmmmm I think that we have different ways to select the responsibilities of objects. 🤔 

I don't understand why we should ask this to the node and not to any Compilation object who already tracks the primitives to compile.